src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -1316,19 +1325,41 @@
   "com.minecolonies.coremod.gui.chat.cancel": "Try to cancel that request!",
   "com.minecolonies.coremod.gui.chat.fulfill": "More details please!",
   "com.minecolonies.coremod.gui.research.title": "Research Branches",
-  "com.minecolonies.coremod.research.requirement.building.level": "Requires %s at least level %d!",
+  "com.minecolonies.coremod.gui.research.tier.header": "Level %d (%d hr)",
+  "com.minecolonies.coremod.gui.research.time": "%s remaining",
+  "com.minecolonies.coremod.gui.research.countinprogress": "%d/%d Research in Progress",
+  "com.minecolonies.coremod.research.research.toomanyinprogress.1": "Too many researches",
+  "com.minecolonies.coremod.research.research.toomanyinprogress.2": "already in progress",
+  "com.minecolonies.coremod.research.research.notenoughresources": "Missing Resources",
+  "com.minecolonies.coremod.research.research.missingrequirements": "Missing Requirements",
+  "com.minecolonies.coremod.research.research.buildingleveltoolow": "Too low University level",
+  "com.minecolonies.coremod.research.requirement.building.level": "Requires %s(s) totaling at least %d levels!",
+  "com.minecolonies.coremod.research.requirement.building.or": " or ",
```

Comment:
you have two "or"s (another on line 1592). Is this intentional?

Reply:
One of them has spaces and is used in hovertext (for now, alternate-building research requirements, but might be relevant for other alternate-x requirements or costs down the road), while the other has no spaces and is used for graphics layout (for now, the onlyChild arrows).

For Romantic languages it's not a critical difference, but there's a few languages where translators are very likely to want to have a different option.  If I understand Finnish properly, `com.minecolonies.coremod.research.requirement.building.or` would probably be better translated as 'tai' (inclusive or), where `com.minecolonies.coremod.research.research.or` would use 'vai' (exclusive or).  Some languages don't use spaces, either at all (Thai) or in some contexts.

That said, if it's more effort than it's worth to maintain, I can collapse the two and call it good enough.  Or I can put them into a different path for generic grammar, if you want one set up.